ROMFS/px4fmu_common/init.d/rc.airship_defaults

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+#
+# Airship default parameters.
+#
+# NOTE: Script variables are declared/initialized/unset in the rcS script.
+#
+
+set VEHICLE_TYPE airship
+
+if [ $AUTOCNF = yes ]
+then
+	param set NAV_ACC_RAD 2
+
+	param set RTL_RETURN_ALT 30
+	param set RTL_DESCEND_ALT 10
+	param set RTL_LAND_DELAY 1
+
+	param set PWM_MAX 2000
```

Comment:
These are the default already, so no need to have them here

Reply:
@dan-leo why did you mark this as resolved? I don't see this being addressed.

Reply:
@bkueng I didn't realize that when defined as a pwm parameter they already have those exact defaults. Updated, thanks!